quadra/datamodules/base.py

```diff
@@ -46,6 +51,49 @@ def __new__(cls, name, bases, dct):
         return super().__new__(cls, name, bases, dct)
 
 
+def compute_file_hash(path: str, hash_size: int) -> str:
```

Comment:
If we cannot put any number for hash size, typing will be confusing. What about using [Literal Typings](https://peps.python.org/pep-0586/) ?

Reply:
I've already used Literal in the datamodule typings actually 😅 Makes sense